### PR TITLE
Single frame model C에 focal loss 적용

### PR DESCRIPTION
**Pull request by keechang-choi**
focal loss 효과 있음
근데 validation set이 작아서 그런지 val loss 작은 버전 보다; 10  epoch 정도 학습 시킨게 테스트 점수가 좋았음
fit-valid-train 비율 및 구조 바꿔서 최종 학습 용 셋 구축 필요

### CODE REVIEW

**Review by dozen12 — COMMENTED** [review]
LGTM

**Review by dozen12 — APPROVED** [review]
LGTM
